src/site/_includes/partials/footer.njk

```diff
@@ -0,0 +1,128 @@
+<footer class="w-footer">
+  <nav class="w-footer__linkboxes">
+    <ul class="w-footer__linkboxes-list">
+      <li class="w-footer__linkbox">
+        <h3 class="w-footer__linkbox-heading">Contribute</h3>
+        <ul class="w-footer__linkbox-list">
+          <li class="w-footer__linkbox-item">
+            <a href="https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=bug&template=bug_report.md&title="
+              class="w-footer__linkbox-link
+            gc-analytics-event" data-category="Site-Wide Custom Events" data-label="Footer Link (index 1)">
```

Comment:
gc-analytics-event seems ignored, we just use data-category and data-label to trigger the event.

Maybe this is an omission in DevSite v2.